src/luxonis_ml/data/dataset.py

```diff
@@ -502,6 +506,47 @@ def _add_process_batch():
                 )
                 self._end_time()
 
+            array_paths = list(
+                set(
+                    [
+                        data["value"]
+                        for data in batch_data
+                        if data["type"] == "array"
+                        or data["type"] == DataLabelType.ARRAY
+                    ]
+                )
+            )
+            if len(array_paths):
+                print("Checking arrays...")
+                self._start_time()
+                data_utils.check_arrays(array_paths)
+                self._end_time()
+                print("Generating array UUIDs...")
```

Comment:
Could we use the new logger instead of prints?

Reply:
Added. Do you mind checking if it's used as intended?